src/interface/widgets/STPVisualizerWidget.cpp

```diff
@@ -107,7 +107,7 @@ namespace rtt::ai::interface {
             parent->setKeeperRole(role, state);
             return;
         }
-        updateContent << "Some(" << botView->getId() << ") => ";
```

Comment:
Maybe make the id more readable somehow since it's only one number behind the role name which also ends with a number